internal/handlers/command.go

```diff
@@ -0,0 +1,192 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handlers
+
+import (
+	"context"
+	"errors"
+
+	"github.com/FerretDB/FerretDB/internal/handlers/common"
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
+	"github.com/FerretDB/FerretDB/internal/wire"
+)
+
+type command struct {
+	name           string
+	help           string
+	handler        func(*Handler, context.Context, *wire.OpMsg) (*wire.OpMsg, error)
+	storageHandler func(common.Storage, context.Context, *wire.OpMsg) (*wire.OpMsg, error)
+}
+
+var commands = map[string]command{
+	"buildinfo": {
```

Comment:
Lst's add a test to ensure that all keys in this map are lowcase

Reply:
Will do, I'll probably also add a test to check the fields (names/help) are unique - because it's going to be easy to copy and paste values here and make mistakes.